doc/contrib/frontend.md

```diff
@@ -78,6 +78,32 @@ Take of example a button component that is used as the login button.
 
 That has the added benefit, that presentational components can be easily integrated into Storybook.
 
+## Documentation
+
+We use Storybook [Storybook](https://storybook.js.org) to document shared components.
+Story files are named `*.stories.tsx` and should be located in the same folder as the component they are documenting.
+For an example please take a look in the `src/components` folder.
+
+For other API documentation like Hooks and utility functions, we use [TypeDoc](https://typedoc.org/).
+To generate the TypeDoc documentation under `src/generated/docs` simply run `npm run typedoc`.
```

Comment:
This is a confusing order in the text: first write the command to execute (I would also mention `npm install`), then write where documentation can be found.